release/resources/module_list.json

```diff
@@ -157,9 +160,27 @@
         {
             "name": "module-ballerinax-candid"
         },
+        {
+            "name": "module-ballerinax-confluent.cavroserdes"
+        },
+        {
+            "name": "module-ballerinax-confluent.cregistry"
+        },
+        {
+            "name": "module-ballerinax-docusign.dsadmin"
+        },
+        {
+            "name": "module-ballerinax-docusign.dsclick"
+        },
+        {
+            "name": "module-ballerinax-docusign.dsesign"
+        },
         {
             "name": "module-ballerinax-github"
         },
+        {
+            "name": "module-ballerinax-googleapis.gcalendar"
```

Comment:
shouldn't this be `module-ballerinax-googleapis.calendar`? connector dashboard seems to be failing due to this.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-library/pull/6320